Vimeo-Native/NugetReferencer/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
-  <package id="VimeoAPI" version="1.0.1.0" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
+  <package id="VimeoAPI" version="1.0.1.0" targetFramework="net4" />
```

Comment:
Actually, I do see the Vimeo API further down the list, it just wasn't indicated as a dependency of the Native Package.